app/views/filters/_form.html.erb

```diff
@@ -1,14 +1,12 @@
-<% javascript 'filters' %>
-
 <%= form_for @filter do |f| %>
   <%= base_errors_for f.object %>
   <ul class="nav nav-tabs" data-tabs="tabs">
     <li class="active"><a href="#primary" data-toggle="tab"><%= _("Filter") %></a></li>
     <% if show_location_tab? %>
-      <li><a href="#locations" data-toggle="tab"><%= _("Locations") %></a></li>
+      <li><a href="#locations" data-toggle="tab" data-allow-locations=true style=<%= display(!@filter.override?) %>><%= _("Locations") %></a></li>
```

Comment:
this shouldn't be `data-allow-locations=true`, the display should depend on whether or not the resource allows location filterings, i.e.:
```suggestion
      <li><a href="#locations" data-toggle="tab" style=<%= display(!(@filter.override? && @filter.allows_location_filtering?))%>><%= _("Locations") %></a></li>
```
same goes for the orgs below